charts/ceph-csi-cephfs/templates/nodeplugin-clusterrole.yaml

```diff
@@ -0,0 +1,19 @@
+{{- if .Values.rbac.create -}}
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: {{ include "ceph-csi-cephfs.nodeplugin.fullname" . }}
+  labels:
+    app: {{ include "ceph-csi-cephfs.name" . }}
+    chart: {{ include "ceph-csi-cephfs.chart" . }}
+    component: {{ .Values.nodeplugin.name }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+    {{- with .Values.commonLabels }}{{ toYaml . | trim | nindent 4 }}{{- end }}
+rules:
+{{- if and .Values.readAffinity .Values.readAffinity.enabled }}
+  - apiGroups: [""]
+    resources: ["nodes"]
+    verbs: ["get"]
```

Comment:
Can you open a new issue to track cephfs encryption support for helm charts?

Reply:
> Can you open a new issue to track cephfs encryption support for helm charts?

https://github.com/ceph/ceph-csi/issues/4267